paket.dependencies

```diff
@@ -1,8 +1,8 @@
 source https://api.nuget.org/v3/index.json
-frameworks: net6.0, netstandard2.0
+framework: net6.0, netstandard2.0
 
 nuget YamlDotNet
-nuget FSharp.Core 4.7.2
+nuget FSharp.Core ~> 5.0.2
```

Comment:
if it is work with  netstandard, you probably can keep `4.7.2` as recommended by official template -  https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/master/templates/content/basic/src/MyProvider.Runtime/MyProvider.Runtime.fsproj#L17

Reply:
Yes, that works too.